cleanup-passes/readme.ts

```diff
@@ -120,7 +120,8 @@ Edit this file, and the bot will squash your changes :)
 ${analyzedElement.desc}
 `;
   }
-  for (const name in behaviorsByName) {
+
+  for (const name of Object.keys(behaviorsByName).sort()) {
```

Comment:
Could the same sort function for `tagNames` be used here? If the repo only has behaviors, it would be nice to have the 'main' one first too.